Detectors/TRD/reconstruction/src/TrackletsParser.cxx

```diff
@@ -305,7 +305,7 @@ int TrackletsParser::Parse()
           // take the header and this data word and build the underlying 64bit tracklet.
           int q0, q1, q2;
           if (mcmtrackletcount > 2) {
-            LOG(alarm) << "mcmtrackletcount is not in [0:2] count=" << mcmtrackletcount << " headertrackletcount=" << headertrackletcount << " something very wrong parsing the TrackletMCMData fields with data of : 0x" << std::hex << *word;
+            LOG(info) << "mcmtrackletcount is not in [0:2] count=" << mcmtrackletcount << " headertrackletcount=" << headertrackletcount << " something very wrong parsing the TrackletMCMData fields with data of : 0x" << std::hex << *word;
```

Comment:
Ok with me to denote this for now, but it should not be forgotten. To me it is not clear how this can happen. Maybe its just a single bad MCM but then we should be able to identify that somehow from our QC

Reply:
it will appear in the qc graphs, and yes I am busy trying to track down the circumstances for it. But it need not be a warning to RC.